<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layer1</class>
 <widget class="QWidget" name="layer1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#layer1{

background-color:qlineargradient(spread:pad, x1:0.147727, y1:0.17, x2:1, y2:1, stop:0 rgba(71, 71, 71, 255), stop:1 rgba(255, 255, 255, 255));
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;; color: white;
</string>
   </property>
   <property name="text">
    <string>Please select a target you want to predict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonToDepressiveness">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(223, 223, 223);
color: rgb(54, 54, 54);
border-radius:10.0px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>Depressiveness</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
